OPHD/States/MapViewStateTurn.cpp

```diff
@@ -50,6 +50,53 @@ static inline void pullFoodFromStructure(FoodProduction* producer, int& remainde
 }
 
 
+static RouteList findRoutes(micropather::MicroPather* solver, TileMap* tilemap, Structure* mine, const StructureList& smelters)
```

Comment:
I agree with re-organizing and grouping `static` freestanding methods.

I think slightly better is to wrap them in an unnamed namespace, which allows the `static` keyword to be removed. This particular use of the `static` keyword is perhaps less well known, and requires more careful reading to distinguish it from static class member methods.

Using unnamed namespaces or `static` on freestanding methods basically does the same thing, in that they prevent access from other translation units. This also protects against potential link errors if the method names are re-used (for different purposes) by separate translation units.

Reply:
I find unnamed namespaces to add more clutter than they're worth. Using `static` in this sense is very simple and anybody who's used C or C++ for more than a year or so should know immediately what it means. On the other hand, link errors due to name collisions is potentially worth it. I guess since the function naming I'm using is very specific I'm not worried about name collisions.